worker/state/manifold.go

```diff
@@ -272,6 +272,11 @@ func newModelStateWorker(
 func (w *modelStateWorker) loop() error {
 	st, err := w.pool.Get(w.modelUUID)
 	if err != nil {
+		if errors.IsNotFound(err) {
```

Comment:
Do we need this?
We will only get NotFound if the state is marked for removal. The reason it's not removed is that the ref count is not yet 0. Calling Remove() below will not affect the ref count, so if effectively a no-op.

Reply:
the change here is to ignore notfound error.

Reply:
right but I am wondering if we need the w.pool.Remove(w.modelUUID)